modules/transactions/src/main/java/org/apache/ignite/internal/tx/LockManager.java

```diff
@@ -55,12 +56,20 @@ public interface LockManager extends EventProducer<LockEvent, LockEventParameter
     void release(UUID txId, LockKey lockKey, LockMode lockMode);
 
     /**
-     * Retrieves all locks for the specified transaction id.
+     * Retrieves all waiters for the specified transaction id.
```

Comment:
Why 'waiters'? In other places 'lock' is used, e.g. `Release all locks associated with a transaction.`